tests/Kiota.Builder.Tests/Plugins/PluginsGenerationServiceTests.cs

```diff
@@ -398,6 +398,21 @@ public static TheoryData<string, string, string, PluginAuthConfiguration, Action
                     Assert.Equal("{oauth2_0_CONFIGURATION_ID}", ((OAuthPluginVault)auth0!).ReferenceId);
                 }
             },
+            // oauth2: implicit
+            {
+                "{securitySchemes: {oauth2_0: {type: oauth2, flows: {implicit: {}}}}}",
+                string.Empty, "security: [oauth2_0: []]", null, resultingManifest =>
+                {
+                    Assert.NotNull(resultingManifest.Document);
+                    Assert.Empty(resultingManifest.Problems);
+                    Assert.NotEmpty(resultingManifest.Document.Runtimes);
+                    var auth0 = resultingManifest.Document.Runtimes[0].Auth;
+                    Assert.IsType<OAuthPluginVault>(auth0);
+                    Assert.Equal(AuthType.OAuthPluginVault, auth0?.Type);
+                    Assert.Equal("{oauth2_0_CONFIGURATION_ID}", ((OAuthPluginVault)auth0!).ReferenceId);
+                }
+            },
```

Comment:
The test for implicit flow should fail. This test data seems to suggest that it succeeds. Wasn't the task to fail on implicit flow?

Reply:
True! I missed that